developmentseed/pearl-frontend
# Allow mosaic change on existing projects

Fix #55.

How to test:

- Open any existing project
- Click mosaic selector button
- Select a different mosaic (after this, retrain should be disabled and existing prediction is cleared)
- Run prediction
- Retrain is enabled
- Reloading project display latest mosaic

@geohacker ready for review.
